Resources/public/css/views/languageswitcher.css

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+.ez-view-languageswitcherview {
+    position: absolute;
+    top: 0;
+    right: 1em;
+    line-height: 2.5em;
+    padding: 0 1em;
+}
+
+.ez-view-languageswitcherview .ez-expandable-area {
+    position: absolute;
+    top: 2.3em;
+    right: 0;
+    overflow-y: auto;
+    display: none;
+    line-height: 1em;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.ez-view-languageswitcherview.is-expanded .ez-expandable-area {
+    display: block;
+}
+
+.ez-view-languageswitcherview .ez-content-translation a {
```

Comment:
please do not style based on tag name to avoid the strong coupling between the CSS and the markup. Instead you should add a class on the links and use it here (and in the theme stylesheet)